hazelcast/src/main/java/com/hazelcast/mapreduce/impl/AbstractJob.java

```diff
@@ -85,7 +86,7 @@ public AbstractJob(String name, JobTracker jobTracker, KeyValueSource<KeyIn, Val
     }
 
     @Override
-    public Job<KeyIn, ValueIn> onKeys(Iterable<KeyIn> keys) {
+    public Job<KeyIn, ValueIn> onKeys(Iterable<? extends KeyIn> keys) {
```

Comment:
This method seems to lack code coverage: https://hazelcast-l337.sonar.cloudbees.com/dashboard/index/12063?display_title=true&metric=coverage